python/cuml/test/test_train_test_split.py

```diff
@@ -343,7 +343,7 @@ def test_stratified_random_seed(seed_type):
     assert y_test.equals(y_test2)
 
     # Ensure that data is shuffled
-    assert not X_train.head().equals(X.head())
+    assert not (X.head().index.values == X_train.head().index.values).all()
```

Comment:
I don't see a test for the issue seen in https://github.com/rapidsai/cuml/pull/2805? Should make sure we include a reproducer.

Reply:
Yeah, I've included now.